src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -360,6 +361,7 @@ public override void GetPEKind(out PortableExecutableKinds peKind, out ImageFile
             return GetMethodInternal(name, bindingAttr, binder, callConvention, types, modifiers);
         }
 
+        [RequiresUnreferencedCode("Methods might be removed")]
```

Comment:
Worth a comment that this operates on the module type, and thus can't be annotated.